<?xml version="1.0" encoding="UTF-8"?>
<<<<<<< HEAD
<testsuite name="be.helha.tests.Test_Parser" tests="4" skipped="0" failures="0" errors="0" timestamp="2022-05-18T09:30:14" hostname="LAPTOP-TBUVHM0E" time="0.037">
  <properties/>
  <testcase name="testPostgres1()" classname="be.helha.tests.Test_Parser" time="0.018"/>
  <testcase name="testPostgres3()" classname="be.helha.tests.Test_Parser" time="0.006"/>
  <testcase name="testPostgres4()" classname="be.helha.tests.Test_Parser" time="0.006"/>
=======
<testsuite name="be.helha.tests.Test_Parser" tests="4" skipped="0" failures="0" errors="0" timestamp="2022-05-18T07:07:17" hostname="LAPTOP-1M6GP7E2" time="0.034">
  <properties/>
  <testcase name="testPostgres1()" classname="be.helha.tests.Test_Parser" time="0.007"/>
  <testcase name="testPostgres3()" classname="be.helha.tests.Test_Parser" time="0.008"/>
  <testcase name="testPostgres4()" classname="be.helha.tests.Test_Parser" time="0.011"/>
>>>>>>> 540d0d3 ( ajout test connection reussi)
  <testcase name="testPostgres5()" classname="be.helha.tests.Test_Parser" time="0.005"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
